viewer/packages/cad-styling/src/prioritized/ClusteredAreaCollection.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { AreaCollection } from './AreaCollection';
+import { SmartMergeBoxes } from './mergeboxes/SmartMergeBoxes';
+import { BoxClusterer } from './mergeboxes/BoxClusterer';
```

Comment:
You don't really need this import - just skip it and omit declaring the type of `_clusterer` below. A bit confusing that there is a interface for this at all - I'd consider just deleting the `BoxClusterer`-interface.

Reply:
You're right, BoxClusterer was supposed to be temporary